examples/investments.jl

```diff
@@ -215,11 +229,16 @@ function generate_example_network_investment()
     ℒᵗʳᵃⁿˢ = [Transmission(𝒜[2], 𝒜[1], [power_line, co2_pipeline])]
 
     # Input data structure
+    # It is also explained on
+    # https://energymodelsx.github.io/EnergyModelsBase.jl/stable/library/public/case_element/
     case = Case(
-        𝒯,
-        𝒫,
-        [𝒩, ℒ, 𝒜, ℒᵗʳᵃⁿˢ],
+        𝒯,                  # Chosen time structure
+        𝒫,                  # Resources used
+        [𝒩, ℒ, 𝒜, ℒᵗʳᵃⁿˢ],  # Nodes, Links, Areas, and Transmission corridors used
         [[get_nodes, get_links], [get_areas, get_transmissions]],
+        # Line above: the first vector corresponds to the original couplings between `Node`said
```

Comment:
Looks like there is a typo here, should it read ...between said `Node` ... something?